types/prosemirror-model/index.d.ts

```diff
@@ -24,6 +24,8 @@ export class ContentMatch {
   findWrappingFor(node: Node): Array<{ type: NodeType, attrs: AnyObject }> | null | undefined;
 }
 export class Fragment {
+  constructor(content: Node[], size: number);
```

Comment:
This [is not public API](https://github.com/ProseMirror/prosemirror-model/commit/802d7ffb8a34a6ebcaa439bbadf146df7656de1a#diff-e0318204de8e4e07839c5509ca81af6cL3), so should not be in the types. (Users should use the static `.from` method instead)